src/styles/FormFields.module.css

```diff
@@ -0,0 +1,9 @@
+.required-label-symbol {
+  color: #b50909;
+  font-weight: 700;
+  text-decoration: none !important;
```

Comment:
Are the !importants necessary?

Reply:
I can't find a way to bypass USWDS baked in styles in another way, but I'm not a CSS wizard.